tests/proj/simple/test_runscript.py

```diff
@@ -1,11 +1,22 @@
 import os
 import pytest
+import torch
 
 import jiant.utils.python.io as py_io
 from jiant.proj.simple import runscript as run
 import jiant.scripts.download_data.runscript as downloader
 
 
+def eq_state_dicts(model_1, model_2):
+    models_differ = 0
+    for key_item_1, key_item_2 in zip(model_1.items(), model_2.items()):
+        if torch.equal(key_item_1[1], key_item_2[1]):
+            pass
+        else:
+            return False
+    return True
```

Comment:
@zphang Do you think this belongs in a utils.py file?

Reply:
Yes, but (1) change the variable names to be generic (`state_dict_1`), and (2) document to emphasize that this is checking the values, not the reference/data pointers.